app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.kt

```diff
@@ -271,7 +267,7 @@ class FeedFragment : BaseStateFragment<FeedState>() {
 
     override fun onDestroyView() {
         // Ensure that all animations are canceled
-        feedBinding.newItemsLoadedButton?.clearAnimation()
+        feedBinding.newItemsLoadedButton.clearAnimation()
```

Comment:
Is this what was fixed in #7648?
Remember to use what is now on `dev` when rebasing